src/Compute/Compute/Az.Compute.psd1

```diff
@@ -57,8 +57,7 @@ DotNetFrameworkVersion = '4.7.2'
 RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.12.2'; })
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = 'Microsoft.Azure.PowerShell.AutoMapper.dll', 
-               'Microsoft.Azure.Management.Compute.dll', 
+RequiredAssemblies = 'AutoMapper.dll', 'Microsoft.Azure.PowerShell.Compute.Sdk.dll', 
```

Comment:
`AutoMapper.dll` is not correct. It should be `Microsoft.Azure.PowerShell.AutoMapper.dll`

Reply:
addressed